querydict.py

```diff
@@ -1,23 +1,5 @@
 
-query_dict = [
-    {
-    "template_id" : "s01",
-    "query_text" : "select * from querylist where template_id = :val"
-    },
-    {
-    "template_id" : "s02",
-    "query_text" : "select * from tbl_user where user_phone = :val"
-    },
-    {
-    "template_id" : "s03",
-    "query_text" : "select * from orders where user_phone = :val"
-    },
-    {
-    "template_id" : "s04",
-    "query_text" : "select * from products where user_phone = :val"
-    },
-    {
-    "template_id" : "s05",
-    "query_text" : "select * from categories where user_phone = :val"
-    }
-]
\ No newline at end of file
+q_template = ["s01","s02","s03","s04","s05"]
+q_query = ["select * from querylist where template_id = :val","select * from tbl_user where user_phone = :val","select * from orders where user_phone = :val","select * from products where user_phone = :val","select * from querylist where template_id = :val",]
```

Comment:
isko query_dict format mein hi kardo. that is better that this format

Reply:
in that format, for finding if template_id exists, 2 loop lagana padega.